src/Illuminate/Cookie/CookieJar.php

```diff
@@ -30,6 +30,13 @@ class CookieJar implements JarContract
     protected $secure = false;
 
     /**
+     * The default httpOnly setting (defaults to true).
+     *
+     * @var bool
+     */
+    protected $httpOnly = true;
```

Comment:
> The default value was true

How can you say that when the changelog clearly shows the opposite.

![screen shot 2016-03-23 at 10 44 27 am](https://cloud.githubusercontent.com/assets/172966/13974156/47ee8ce4-f0e4-11e5-9fbd-785fc4f5b522.png)